scripts/UpdateNugetCredentials.ps1

```diff
@@ -0,0 +1,8 @@
+param([Parameter(Mandatory=$true)][string]$username, [Parameter(Mandatory=$true)][string]$apiToken, [Parameter(Mandatory=$true)][string]$nugetFileAbsolutePath)
```

Comment:
maybe add a comment to say the original copy is on kiota and if any update is done, they should be synced?

Reply:
Resolved via https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/pull/333/commits/8a8ac8eb95d445c4c5d6af93f3f326a69a9cf77c